frontends/web/src/locales/en/app.json

```diff
@@ -26,6 +26,7 @@
     "extendedPublicKey": "Extended public key",
     "label": "Account info",
     "scriptType": "Script type",
+    "taproot": "The Taproot extended public key is not shown to protect from accidental misuse, as raw formats for this script type are not widely supported. Expert users who want to view this key should connect their device to wallet software that supports the output descriptor format.",
```

Comment:
i recently read somewhere that it's not good to call users users :smile:  Maybe "If you want to view this key ..."?